### PR TITLE
docs: adds note about dynamic config

### PR DESCRIPTION
**Pull request by jritsema**
I ran into an issue with this and wanted to help surface up and clarify which sections are supported.

### CODE REVIEW

File: docs/docs/configuration/index.md [hunk 1]
```diff
@@ -37,6 +37,8 @@ It is not recommended to copy this full configuration file. Only specify values
 
 :::
 
+**Note:**  Dynamic configuration values are currently supported in the following sections: `mqtt.password` and `cameras.<name>.ffmpeg.inputs.[].path`.  Dynamic values can be set using environment variables with names starting with `FRIGATE_`.  For example, `{FRIGATE_MQTT_PASSWORD}`, `{FRIGATE_RTSP_USER}`, `{FRIGATE_RTSP_PASSWORD}`.
```
Comment (NickM-27) [inline docs/docs/configuration/index.md:40]: Definitely a good idea to call out, but I don't think there is a reason to be as technical. Can just say it is only supported for MQTT Password and ffmpeg paths

**Review by NickM-27 — APPROVED** [review]

**Review by blakeblackshear — CHANGES_REQUESTED** [review]
I don't like the term "Dynamic configuration values". I would just state that "Environment variable substitution is only supported..." and "Only environment variables starting with FRIGATE_ will be used".

File: docs/docs/configuration/index.md [hunk 2]
```diff
@@ -37,6 +37,8 @@ It is not recommended to copy this full configuration file. Only specify values
 
 :::
 
+**Note:**  Dynamic configuration values are currently only supported for MQTT password and camera ffmpeg paths.  Dynamic values can be set using environment variables with names starting with `FRIGATE_`.  For example, `{FRIGATE_MQTT_PASSWORD}`, `{FRIGATE_RTSP_USER}`, `{FRIGATE_RTSP_PASSWORD}`.
```
Comment (felipecrs) [inline docs/docs/configuration/index.md:40]:
```suggestion
**Note:**  Dynamic configuration values are currently only supported for MQTT password and camera ffmpeg paths. Dynamic values can be set using environment variables with names starting with `FRIGATE_`.  For example, `{FRIGATE_MQTT_PASSWORD}`, `{FRIGATE_RTSP_USER}`, `{FRIGATE_RTSP_PASSWORD}`.
```

**Comment by jritsema** [comment]
Last commit addresses @blakeblackshear's [comment](https://github.com/blakeblackshear/frigate/pull/4882#pullrequestreview-1235268572).

**Review by blakeblackshear — APPROVED** [review]

**Pull request merged**